base62.py

```diff
@@ -27,6 +43,17 @@ def encode(n):
 
     return ''.join(s)
 
+
+def encodebytes(s):
+    """Encode a bytestring into a bytes object containing multiple lines
```

Comment:
In Python 3 `encode` and therefore `encodebytes` returns a str rather than bytes.